projects/nx-verdaccio/src/executors/env-bootstrap/verdaccio-registry.ts

```diff
@@ -165,3 +173,26 @@ export async function startVerdaccioServer({
     throw error;
   }
 }
+
+export function stopVerdaccioServer(options: {
+  projectName: string;
+  verbose?: boolean;
+  configuration?: string;
+  environmentRoot: string;
+  context: ExecutorContext;
+}): Promise<void> {
```

Comment:
@BioPhoton LMK if you would rather have a standard signature

Reply:
What you think is best.

Reply:
Maybe a standard will emerge with all the teardown functions...